<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>731</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>956</width>
    <height>731</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Client viewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow
{
	background-color: rgb(36, 36, 36);
	border:2px solid rgb(56, 56, 56);
	border-left:2px solid rgb(56, 56, 56);
}

QWidget
{
	font: 10pt &quot;Open Sans&quot;;
	color: white; 
	background-color: rgb(66, 66, 66);
	selection-background-color :rgb(64, 167, 195);
}

QFrame#AcquisitionParameters, QFrame#FilteringParameters, QFrame#Diagnosis, QFrame#Diagnosis_2, QFrame#EntryFile_2, QFrame#SaveFile, QFrame#SaveFileWavefront {
	background-color: rgb(36, 36, 36);
}

QWidget#currentFile, QWidget#refFile, QWidget#Mask, QWidget#maskLoad {
	background-color: rgb(36, 36, 36);
}

QMainWindow#mainWindow
{
	margin:5px 0px 5px 0px;
}

/*toolbars*/


QToolBar {
	border: none;
	margin:2px;
}

QToolBar {
	background-color: rgb(64, 167, 195);
	padding: 0px 0px 0px 5px;
	margin-bottom:5px;
}

QToolBar QWidget {
	background-color: rgb(64, 167, 195);
}

QMessageBox {
	background-color: rgb(56, 56, 56);
	border-bottom: 2px solid rgb(66, 66, 66);
	border-left: 2px solid rgb(66, 66, 66);
	border-right: 2px solid rgb(66, 66, 66);
}

/* Push button */

QPushButton
{
	background-color: rgb(56, 56, 56);
}

QPushButton:disabled
{
	background-color: rgb(96, 96, 96);
}

QPushButton:checked
{
	background-color: rgb(64, 167, 195);
}

QFrame QPushButton
{
	background-color: rgb(96,96, 96);
	border: 0px solid white;
}

/* Label */

QLabel
{
	background-color: transparent;
}

/* Tool Tip */

QToolTip 
{ 
	border: 1px solid rgb(64, 167, 195);
}

/* Checkbox */

QCheckBox
{
	background-color:transparent;
	color: rgb(255, 255, 255);
}

QCheckBox::indicator
{
	border: 1px solid white;
	background-color: rgb(56, 56, 56);
}

QCheckBox::indicator:checked 
{
	background-color:rgb(64, 167, 195);
}

/* Combobox */

QComboBox
{
	background-color: rgb(56, 56, 56);
	border: 1px solid rgb(66,66,66);
}

/* Line edit */

QLineEdit
{
	background-color: rgb(56, 56, 56);
	border: 1px solid rgb(96, 96, 96);
}

/* Spinbox */

QDoubleSpinBox, QSpinBox
{
	font: 10pt &quot;Open Sans&quot;;
	background-color: rgb(56, 56, 56);
	border: 1px solid rgb(66,66,66);
	padding-right: 15px;
}

QDoubleSpinBox:disabled, QSpinBox:disabled
{
	border: 0px solid rgb(66,66,66);
	padding-right: 5px;
}

QDoubleSpinBox::up-button:disabled, QSpinBox::up-button:disabled, 
QDoubleSpinBox::down-button:disabled, QSpinBox::down-button:disabled
{
	width: 0px;
	height: 0px;
}

QTabWidget::pane { 
    border: none;
	background-color: rgb(56, 56, 56);
}

QTabBar::tab {
    background-color: rgb(66, 66, 66);
    border: none;
	padding:5px;
    padding-right: 25px;
    min-width: 40ex;
	margin-right:2px;
	margin-left:2px;
}

QPushButton:enabled {
    background-color:#ffffff;
    color: black;
}

QPushButton#clientStatus:enabled, QPushButton#serverStatus:enabled {
    background-color: green;
}

QPushButton#clientStatus:disabled, QPushButton#serverStatus:disabled {
    background-color: red;
}

QTabBar::tab:selected {
	color: white;
	font: bold 12pt &quot;Open Sans&quot;;
	border : 1px solid white;
}

QTabBar::tab:!selected {
	color: rgb(64, 167, 195);
	font: 12pt &quot;Open Sans&quot;;
	border : 1px solid rgb(64, 167, 195);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QFrame" name="serverFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Server</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="serverStatus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ServerStartButton">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ServerStopButton">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Client</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clientStatus">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClientStartButton">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClientStopButton">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="clientFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="ImageDisplay" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/WaveView/icon/Play</normaloff>
     <normalon>:/WaveView/icon/Pause</normalon>:/WaveView/icon/Play</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
